### ISSUE TITLE
Sim iFrame shift when tab changes focus in a11y-view html

### ISSUE THREAD

**Issue by mbarlow12** [phetsims/chipper#687]
I just released a new one-off version of Faraday's Law and noticed that when you tab around the a11y-view, the sim jumps up with each press until the position below.

![image](https://user-images.githubusercontent.com/5711147/42253071-3e1e051a-7efd-11e8-91f2-9f1f24376e87.png)

Not sure about who's the responsible dev for this, so I tagged you both, @zepumph @jessegreenberg.

**Comment by zepumph** [comment]
I see this as a potential for the version you were finding the bug in, 
https://phet-dev.colorado.edu/html/faradays-law/1.4.0-jump.2/phet/faradays-law_a11y_view.html

I was not able to reproduce this problem. By your brief description above, I imagine that it was easy and noticeable when it was happening. I also couldn't see it in master.

Could you please do the following things on multiple browsers if possible as maybe it is a weird browser thing:
1. reproduce it again from the jump version.
2. is it reproducible from master?
3. If possible, is it reproducible from a previous jump one-off?

Also what is your OS? Are there any errors in the console?

Good Luck!

(p.s. the jumping is so cool! Good work)

**Comment by jessegreenberg** [comment]
Weird! I am also not able to reproduce in master or in https://phet-dev.colorado.edu/html/faradays-law/1.4.0-jump.2/phet/faradays-law_a11y_view.html, where this bug was also noted in https://github.com/phetsims/faradays-law/issues/109.

The focusable items are shifted and stacked vertically along the side of the simulation, it looks like the browser is trying to scroll down to those items when they receive focus.

**Comment by zepumph** [comment]
It would also be good, if you can reproduce, to test if it occurs in the normal iframe sim, https://phet-dev.colorado.edu/html/faradays-law/1.4.0-jump.2/phet/faradays-law_all_iframe_phet.html

**Comment by mbarlow12** [comment]
Hmm - this might be specific to FF on Arch. I'll investigate further.

**Comment by jessegreenberg** [comment]
Might just be Firefox, we can  see it again here, outside of the a11y view:
https://github.com/phetsims/resistance-in-a-wire/issues/160

**Comment by jessegreenberg** [comment]
@mbarlow12 thanks for pointing this out, this was indeed a bug on all Firefox. I think it is fixed in master, can you please verify?

**Comment by mbarlow12** [comment]
I noticed that it is related to PDOM content. When I remove all a11y content from a Node/AccessibleSectionNode/etc... the bug no longer occurs.

**Comment by jessegreenberg** [comment]
That makes sense since the PDOM would be short enough to fit in the Display bounds.

**Comment by mbarlow12** [comment]
I'm seeing it Ohm's Law and RIAW in and out of the a11y view.

**Comment by jessegreenberg** [comment]
Here is the issue related to this, that introduced the regression: https://github.com/phetsims/scenery/issues/822

**Comment by jessegreenberg** [comment]
On slack @mbarlow12 said
> I hadn't realized it was scenery to pull. Yes, fixed.

My fault for not indicating the fix was located in scenery in  https://github.com/phetsims/chipper/issues/687#issuecomment-402874648. But this issue can be closed.
